ja/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <title>「質問していいですか」と聞かないで</title>
+  <meta property="og:title" content="「質問していいですか」と聞かないで" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com/ja" />
+  <meta property="og:locale" content="ja_JP" />
+  <meta property="og:image" content="https://dontasktoask.com/favicon.png"/>
+  <link rel="icon" href="/favicon.png" type="image/png" />
+  <link rel="stylesheet" type="text/css" href="/style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@400;700">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <style>
+    body {
+      font-family: "Noto Sans JP" !important;
+    }
+    em {
+      font-style: normal;
+      font-weight: 700;
+    }
+  </style>
+</head>
+<body>
+  <main>
+    <h1>「質問していいですか」と聞かないで</h1>
+    <p>
+      私がよく見ているオンラインのチャットルームでは，ときどき誰かが入ってきてこのようなことを言います．
+    </p>
+    <blockquote>
+      <span class="name">Hogehoge123:</span>
+      <p class="message">
+        Java 分かる人いますか？
```

Comment:
```suggestion
        Java分かる人いますか？
```
変に空白開けない方がチャット感が出ると思いました。（個人の意見です）